content/docs/components-and-props.md

```diff
@@ -16,23 +16,24 @@ prev: rendering-elements.html
 next: state-and-lifecycle.html
 ---
 
-Components let you split the UI into independent, reusable pieces, and think about each piece in isolation. This page provides an introduction to the idea of components. You can find a [detailed component API reference here](/docs/react-component.html).
+componentहरूबाट तपाईंले UI लाई स्वतन्त्रा रूपमा  विभाजन  गर्न , पुन: प्रयोज्य टुक्राहरूमा विभाजन गर्न  र प्रत्येक टुक्रालाई अलग अलग  रूपमा सोच्न सक्नुहुन्छ।यस पृष्ठले componentsको विचारलाई  परिचय प्रदान गर्दछ। तपाईंले यो linkमा क्लिक गर्नुभयो भने  componet APIको बारेमा विस्तृत  जानकारी प्राप्त गर्न सक्नुहुन्छ [detailed component API reference here](/docs/react-component.html).
 
-Conceptually, components are like JavaScript functions. They accept arbitrary inputs (called "props") and return React elements describing what should appear on the screen.
+अवधारणागत रूपमा, componentहरू JavaScript functions जस्तै हुन्। तिनीहरू मध्यस्थ इनपुट  (“props” भनिन्छ) लिन्छन् र स्क्रिनमा के देखा पर्दछ भनेर वर्णन गर्ने Reactको elements फिर्ता गर्दछन्।
```

Comment:
I didn't quite understand `तिनीहरू मध्यस्थ इनपुट  (“props” भनिन्छ) लिन्छन्`

Should it be: `तिनीहरूले इनपुट(जसलाई “props” भनिन्छ) लिन्छन्`?